DiscreetORM.ts

```diff
@@ -86,13 +86,11 @@ export class StoredClass implements ObjectListener<any>{
 
     createNewTable(obj: any) : void {
         let table_name = obj.constructor.name;
-        let command = "CREATE TABLE " + table_name + "( ";
-        command += 'orm_id INT(255), '
-		for (let attribute of Object.keys(obj)){
-            let attribute_type = obj[attribute].constructor.name;
-            command += attribute + " " +  this.tsTypeToSQLType(attribute_type) + ", ";
-        }
-        command += ")";
+        let command = "CREATE TABLE " + table_name + "(";
```

Comment:
we probably should make sure these strings get escaped but that's a problem for later